src/Tribe/Template/Month.php

```diff
@@ -469,8 +468,8 @@ private function get_event_ids_by_day( $date ) {
 						}
 					}
 
-					$event_start = strtotime( tribe_get_start_date( $event->ID ) );
-					$event_end   = strtotime( tribe_get_end_date( $event->ID ) );
+					$event_start = strtotime( tribe_get_start_date( $event->ID, true, 'Y-m-d H:i:s' ) );
+					$event_end   = strtotime( tribe_get_end_date( $event->ID, true, 'Y-m-d H:i:s' ) );
```

Comment:
This is just an observation - personally, I think a string literal is far more readable here - but "should" we use the relevant date utils constant?